gyp_xwalk

```diff
@@ -354,6 +354,18 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  parameter = '-Dverify_xwalk_apk='
```

Comment:
You'd still need to declare a default value first I think:

```python
'variables': {
  # ...
  'verify_xwalk_apk%: 0,
},
'configurations': {
  # ...
  # Likely use 'verify_xwalk_apk%' instead of 'verify_xwalk_apk'
},
```

Why do you think it makes sense to distinguish release and debug in this case?